packages/rx_bloc_cli/lib/src/processors/ios/podfile_processor.dart

```diff
@@ -10,7 +11,7 @@ class PodfileProcessor extends StringProcessor {
   /// Podfile file processor constructor
   PodfileProcessor(super.args);
 
-  static const _minSupportedIOSVersion = '13.0';
+  static const _minSupportedIOSVersion = '16.0';
```

Comment:
Make sure such important changes are communicated in the changelog

Reply:
Done